src/settings/mod.rs

```diff
@@ -12,6 +12,7 @@ use globset::{Glob, GlobMatcher, GlobSet};
 use itertools::Either::{Left, Right};
 use itertools::Itertools;
 use once_cell::sync::Lazy;
+#[cfg(test)]
```

Comment:
Is this intended?

Reply:
Yes the import is only used in the test-only `for_rule`(s) functions and omitting this attribute results in clippy complaining about an unused import. An alternative would be to move the import into the functions but I don't really see the benefit.